src/js/main.js

```diff
@@ -228,24 +228,24 @@ const removeGrid = () => {
 
 const changeCells = (cells) => {
   const playerTurnDiv = document.querySelector('#player_turn');
-  setPlayerTurn(playerTurnDiv, game.getCurrentPlayer().name);
+  setPlayerTurn(playerTurnDiv, Game.getCurrentPlayer().name);
   cells.forEach((cell) => cell.addEventListener('click', function () {
-    if (game.getBoard().setCell(this.id, game.getCurrentPlayer().symbol)) {
-      this.innerHTML = getSymbol(game.getCurrentPlayer().symbol);
-      const gameOver = game.gameOver();
+    if (Game.getBoard().setCell(this.id, Game.getCurrentPlayer().symbol)) {
```

Comment:
on line 232, my editor had a warning, expecting an arrow function there , to something like this :
```
 cells.forEach((cell) => cell.addEventListener('click', () => {
```